bundle/edu.gemini.shared.skyobject/src/main/java/edu/gemini/shared/skyobject/Magnitude.java

```diff
@@ -235,7 +228,7 @@ public int hashCode() {
      */
     @Override
     public int compareTo(Magnitude that) {
-        int res = system.compareTo(that.system);
+        int res = MagnitudeSystem.all().indexOf(that.system) - MagnitudeSystem.all().indexOf(this.system);
```

Comment:
I agree that there is no meaningful order for the systems and this is entirely arbitrary. My focus here was on mimicking the old behavior in order to avoid unintended side-effects if the order suddenly changes.